.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js 
+      uses: actions/setup-node@v1  
+
+    - name: Installing Dredd
+      run: sudo npm install dredd --global --unsafe-perm=true --allow-root
+
+    - name: Installing node-fetch module
+      run: npm install node-fetch@2
+
+    - name: Installing hooks module  
+      run: npm install hooks
+
+    - name: Installing https module   
+      run: npm install https
+
+    - name: Installing file module 
+      run: npm install fs
```

Comment:
I recommend making a package.json, doing npm install and committing a .lock file, or else this will be a moving target and will break continuously.

Reply:
Added package.json and .lock file for all node modules in the commit [f58fc02](https://github.com/opensearch-project/opensearch-api-specification/pull/35/commits/f58fc02390f48e68ee7237a891c95c7224d0a24e) except the dredd module because of it's broken npm install. I have reached out to the dredd team for the same https://github.com/apiaryio/dredd/issues/921.